scss/components/_article.scss

```diff
@@ -0,0 +1,223 @@
+.article{
+    padding: 0 20px;
+    #article-container{
+        & > div:first-child{
+            max-width: 1200px;
+            margin: 0 auto;
+            padding: 60px 0px 190px;
+            display: flex;
+            flex-direction: column;
```

Comment:
There are a few instances of an element being set to `display: flex; flex-direction: column` where I don't think it's needed. If a layout is going to be a column in both desktop and mobile, there's no reason to make it flex. Just let the block structure do its job. Please look for examples of this in all SCSS files and make changes accordingly.